Appier.xcframework/Info.plist

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>AvailableLibraries</key>
+	<array>
+		<dict>
+			<key>LibraryIdentifier</key>
+			<string>ios-i386_x86_64-simulator</string>
+			<key>LibraryPath</key>
+			<string>Appier.framework</string>
```

Comment:
Should it be xcframework?

Reply:
XCFramework is the container, but it still contains inside a .framework for each architecture.